src/SearchGateway/Model/PrimoSilo.php

```diff
@@ -7,11 +7,11 @@
  */
 Class PrimoSilo extends Silo {
 
-  public function __construct($primoHost, $primoKey, $primoBook, $vid) {
+  public function __construct($primoHost, $primoKey, $option, $vid) {
```

Comment:
I think I'd like to reorder the method signature here so that `$option` comes last.